src/Extractor/BaseExtractor.php

```diff
@@ -96,8 +100,16 @@ public function getGetTablesMetadataSerializer(): GetTablesSerializer
 
     public function getTables(): array
     {
+        $loadColumns = $this->parameters['tableListFilter']['listColumns'] ?? true;
```

Comment:
Niektore DB extractory pouziva nacitanie `getTables` v 2 krokoch, ... najprv iba tabulky a potom stlpce z kontretnej tabulke. V `db-extractor-common` je na to podpora vo vsetkych vrstvach, no chybalo toto nacitanie konfiguracie. .... Metodu `getTables` som musel v extractoroch, ktorych sa to tyka, vzdy prepisovat a pridavat to tam .... tak som to dal radsej tu:
- pre extractory, ktore to nepodporuju to nie je ziadna zmena, .... nemaju to v configu a plati pre nich defaultne spravanie sa ... tj. nacita sa vsetko
- pre extractory, ktore podporuju 2-krokove nacitanie, .... nebudem musite zbytocne prepisovat metodu `getTables`

Pri buducom refactoringu konfiguracie sa tento kód presunie do `Config` objektu.